### PR TITLE
Fix broken components

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

File: components/python/coherence/manifests/sample-manifest.p5m [hunk 1]
```diff
@@ -28,7 +28,6 @@ file path=usr/lib/python2.7/vendor-packages/Coherence-$(COMPONENT_VERSION)-py2.7
 file path=usr/lib/python2.7/vendor-packages/Coherence-$(COMPONENT_VERSION)-py2.7.egg-info/SOURCES.txt
 file path=usr/lib/python2.7/vendor-packages/Coherence-$(COMPONENT_VERSION)-py2.7.egg-info/dependency_links.txt
 file path=usr/lib/python2.7/vendor-packages/Coherence-$(COMPONENT_VERSION)-py2.7.egg-info/entry_points.txt
-file path=usr/lib/python2.7/vendor-packages/Coherence-$(COMPONENT_VERSION)-py2.7.egg-info/requires.txt
```
Comment (pyhalov) [inline components/python/coherence/manifests/sample-manifest.p5m]: Don't see this change here

**Comment by alarcher** [comment]
@pyhalov Weird... I checked it three times, still missing. Anyway removed for now.

**Review by AndWac — COMMENTED** [review]
I say: ship it.
Only two nitpicks:
I find it strange that secure hashes have changed for libdaq and libgit2.
I have a different auto-generated package dependency:
# Auto-generated contents below.  Please manually verify and remove this comment
REQUIRED_PACKAGES += SUNWcs
REQUIRED_PACKAGES += diagnostic/scanpci
REQUIRED_PACKAGES += library/graphics/pixman
REQUIRED_PACKAGES += service/hal
REQUIRED_PACKAGES += system/library
REQUIRED_PACKAGES += system/library/libdbus
REQUIRED_PACKAGES += system/library/math
REQUIRED_PACKAGES += x11/library/libdmx
REQUIRED_PACKAGES += x11/library/libdrm
REQUIRED_PACKAGES += x11/library/libx11
REQUIRED_PACKAGES += x11/library/libxau
REQUIRED_PACKAGES += x11/library/libxcb
REQUIRED_PACKAGES += x11/library/libxdmcp
REQUIRED_PACKAGES += x11/library/libxext
REQUIRED_PACKAGES += x11/library/libxfixes
REQUIRED_PACKAGES += x11/library/libxfont2
REQUIRED_PACKAGES += x11/library/libxi
REQUIRED_PACKAGES += x11/library/libxmu
REQUIRED_PACKAGES += x11/library/libxmuu
REQUIRED_PACKAGES += x11/library/libxpm
REQUIRED_PACKAGES += x11/library/libxrender
REQUIRED_PACKAGES += x11/library/libxshmfence

**Review by AndWac — APPROVED** [review]
I am not experienced enough to check all the changes.
The ones I checked were ok (despite xorg-server).

**Pull request merged**